.github/workflows/release.yaml

```diff
@@ -22,6 +22,7 @@ jobs:
         run: |
           docker login -u="${{ secrets.QUAY_BOT }}" -p="${{ secrets.QUAY_PASSWORD }}" quay.io
           export IMG_TAG="${{ steps.get_release.outputs.tag_name }}"
+          export VALIDAOTR_IMAGE_TAG="${{ steps.get_release.outputs.tag_name }}"
```

Comment:
Typo in `VALIDAOTR_IMAGE_TAG`

Reply:
Also where is this env being used?

Reply:
The typo was in the `Makefile` itself, fixed at both places now

Reply:
This exported tag will be used in the `Makefile` to tag the image that was built from the release